### ISSUE TITLE
Websocket and SSH connections times out

### ISSUE THREAD

**Issue by helnokaly** [Jigsaw-Code/outline-client#631]
**Describe the bug**
Websocket and SSH connections times out in 30 seconds

**To Reproduce**
Steps to reproduce the behavior (Websocket):
1. Go to http://www.websocket.org/echo.html
2. Click on the 'Connect' button
3. Wait 30 seconds
4. See 'Disconnected' message
![image](https://user-images.githubusercontent.com/15964842/52044412-cdad3900-254a-11e9-92f9-60e7686e7db9.png)

Steps to reproduce the behavior (SSH):
1. Connect to a VPS using SSH
2. Wait 30 seconds
4. See 'Connection closed' message
![screenshot from 2019-01-22 21-10-02](https://user-images.githubusercontent.com/15964842/51559600-fff3d200-1e8a-11e9-90d0-2085fc58083d.png)

**Expected behavior**
SSH and websocket connections should no time out

**Desktop (please complete the following information):**
 - OS: Linux (Ubuntu 18.04)

**Additional context**
Duplicate of https://github.com/Jigsaw-Code/outline-server/issues/85
Confirmed by @fortuna on https://github.com/Jigsaw-Code/outline-server/issues/85#issuecomment-498804252
```
Thank you all.
It seems that this is an issue specific to our Electron-based clients: Windows and Linux.

I checked whether we were disabling the timeout for ss-local, but it seems we are:
https://github.com/Jigsaw-Code/outline-client/blob/91a46da368f35faffa4b81efa6919fd334206224/src/electron/process_manager.ts#L340

@alalamav any ideas? Maybe it's something to do with tun2socks?
```

**Comment by alalamav** [comment]
Hi @helnokaly, we are working on a fix (#634) to this issue. shadowsocks-libev v3.3.0 disables client timeouts and I have verified that with this update ssh and websockets do not timeout. I'll follow up once the fix has been merged.

**Comment by fortuna** [comment]
Fixed and released. See https://github.com/Jigsaw-Code/outline-server/issues/85
